Functions.Templates/Templates/SqlOutputBinding-Python/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import azure.functions as func
+import collections
+
+def main(req: func.HttpRequest, product: func.Out[func.SqlRow]) -> func.HttpResponse:
```

Comment:
`// Visit https://aka.ms/sqlbindingsoutput to learn how to use this output binding`